src/DefaultBuilder/test/Microsoft.AspNetCore.NativeAotTests/Microsoft.AspNetCore.NativeAotTests.proj

```diff
@@ -0,0 +1,9 @@
+<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
```

Comment:
I assume this is required because of the fact that this is not an SDK project. I know you are just copying what we do in dotnet/runtime, but I wonder if instead we should just use the NoTargets SDK instead that way these imports are automatic.

Reply:
Check out my latest commit. I tried using NoTargets and Traversal, but ended up being a normal Microsoft.NET.Sdk project. I needed to do something different in order to get the `ExcludeFromBuild` arcade logic to work correctly. The way aspnetcore's build works is a lot different than how dotnet/runtime's build works. runtime is more "top-down" and aspnetcore is more "bottom-up".